<launch>
    <arg name="imu_topic" default="/camera/gyro_accel/sample"/>
    <arg name="imu_filtered_topic" default="/imu_filtered"/>
    <arg name="max_store" default="50"/>
    <node name="avg_imu" pkg="depth_plane_est" type="avg_imu" output="screen">
        <param name="imu_topic" type="string" value="$(arg imu_topic)"/>
        <param name="imu_filtered_topic" type="string" value="$(arg imu_filtered_topic)"/>
        <param name="max_store" type="int" value="$(arg max_store)"/>
    </node>
    
<!--
  <param name="depth_img_topic" value="/camera/depth/image_raw"/>
  <param name="depth_intrinsic_topic" value="/camera/depth/camera_info"/>
  <param name="yaml_file_path" value="/catkin_ws/src/gravity_plane_est/yaml/find_planes.yaml"/>
  <node name="find_planes" pkg="gravity_plane_est" type="find_planes" output="screen"/>
-->
</launch>